index.bs

```diff
@@ -1,17 +1,83 @@
-Abstract: A short description of your spec, one or two sentences.
+URL: https://wicg.github.io/storage-buckets/
+Editor: Evan Stade, Google https://www.google.com/, estade@google.com
+Editor: Ayu Ishii, Google https://www.google.com/, ayui@google.com
+Former Editor: Victor Costan
+!Participate: <a href="https://github.com/WICG/storage-buckets">GitHub WICG/storage-buckets</a> (<a href="https://github.com/WICG/storage-buckets/issues/new">new issue</a>, <a href="https://github.com/WICG/storage-buckets/issues?state=open">open issues</a>)
+Abstract: The Storage Buckets API provides a way for sites to organize locally stored data into groupings called "storage buckets". This allows the user agent or sites to manage and delete buckets independently rather than applying the same treatment to all the data from a single origin.
 </pre>
 
-Introduction {#intro}
-=====================
+<h2 id="storage-bucket-manager">The {{StorageBucketManager}} interface</h2>
 
-See https://github.com/tabatkins/bikeshed to get started.
+<xmp class="idl">
+[SecureContext]
+interface mixin NavigatorStorageBuckets {
+  [SameObject] readonly attribute StorageBucketManager storageBuckets;
+};
+Navigator includes NavigatorStorageBuckets;
+WorkerNavigator includes NavigatorStorageBuckets;
+</xmp>
+
+Each [=environment settings object=] has an associated {{StorageBucketManager}} object.
+
+The <dfn attribute for=NavigatorStorageBuckets><code>storageBuckets</code></dfn>
+getter steps are to return [=this=]'s {{StorageBucketManager}} object.
+
+<xmp class="idl">
+[Exposed=(Window,Worker),
+ SecureContext]
+interface StorageBucketManager {
+    Promise<StorageBucket> open(DOMString name, optional StorageBucketOptions options = {});
+    Promise<sequence<DOMString>> keys();
+    Promise<undefined> delete(DOMString name);
+};
+
+enum StorageBucketDurability {
+  "strict",
+  "relaxed"
+};
+
+dictionary StorageBucketOptions {
+  boolean? persisted = null;
+  StorageBucketDurability? durability = null;
+  unsigned long long? quota = null;
+  DOMHighResTimestamp? expires = null;
+};
+</xmp>
+
+<h3 id="storage-bucket-open">Creating a bucket</h3>
+<h3 id="storage-bucket-delete">Deleting a bucket</h3>
+<h3 id="storage-bucket-keys">Enumerating buckets</h3>
+
+<h2 id="storage-bucket">The {{StorageBucket}} interface</h2>
+<xmp class="idl">
+[Exposed=(Window,Worker),
+ SecureContext]
+interface StorageBucket {
+  [Exposed=Window] Promise<boolean> persist();
+  Promise<boolean> persisted();
+
+  Promise<StorageEstimate> estimate();
+
+  Promise<StorageBucketDurability> durability();
+
+  Promise<undefined> setExpires(DOMHighResTimeStamp expires);
+  Promise<DOMHighResTimeStamp?> expires();
+
+  [SameObject] readonly attribute IDBFactory indexedDB;
+
+  [SameObject] readonly attribute LockManager locks;
```

Comment:
Also can we leave this off for now, and add it in a separate PR? I'm think it's worth debating whether this adds value.

Reply:
Makes sense, removed.